openml/testing.py

```diff
@@ -56,7 +56,7 @@ class TestBase(unittest.TestCase):
     logger = logging.getLogger("unit_tests_published_entities")
     logger.setLevel(logging.DEBUG)
 
-    def setUp(self, n_levels: int = 1) -> None:
+    def setUp(self, n_levels: int = 1, tmpdir_suffix: str = "") -> None:
```

Comment:
Required for running tests with the same name concurrently (this happens with pytest parametrize)